internal/store/organizations.go

```diff
@@ -45,6 +45,10 @@ type OrganizationDeleter interface {
 	DeleteOrganization(string) error
 }
 
+type OrganizationInvitationDeleter interface {
+	DeleteInvitation(string, string) error
+}
+
```

Comment:
[q] I seem to remember a conversation previously that Organizations was getting quite large and breaking out the invitations commands to their own go package. Where did that land/should this endpoint be in a more self contained file?

Reply:
I believe you are referring to this [comment](https://github.com/mongodb/go-client-mongodb-atlas/pull/227#discussion_r659768865) from Gustavo in the atlas go client. I like the idea of splitting the file here as well to keep it as small as possible. I am going to do it in the next commit.